app/routes/application.js

```diff
@@ -6,9 +6,12 @@ export default class ApplicationRoute extends Route {
   @service session;
   @service role;
   @service router;
+  @service intl;
 
   async beforeModel() {
     await this.session.setup();
+    this.intl.setLocale(['nl-BE']);
```

Comment:
This works around an issue with ember-intl v6, but it's good to do anyways since we want the editor labels to be in Dutch.